cb/cb.py

```diff
@@ -80,9 +80,10 @@ def __call__(self, *args):
             if select_as_int in range(len(inputs)):
                 self.out = args[select_as_int]
                 return self.out
-            # TODO(raj): Handle the case where we select the constant value or
```

Comment:
We should eventually refactor this interface to match the idea that we expose high level functions rather than bit-level functions in the functional model. So, we can configure specific things (like mux sel vs constant val). I can do this refactoring in a separate commit.